Reportedly/Utilities/LocalNotificationScheduler.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  LocalNotificationScheduler.swift
+//  Reportedly
+//
+//  Created by Minh Pham on 09/12/2020.
+//
+
+import Foundation
+import UIKit
+import UserNotifications
+
+final class LocalNotificationScheduler {
+    
+    static let shared = LocalNotificationScheduler()
+    
+    func setupDailyStandupLocalNotifications() {
+        // Create the notification content
+        let content = UNMutableNotificationContent()
+        content.title = "Daily Standup Report"
+        content.body = "Hi, it's time to start the daily standup. Please, open the app and answer your daily questions"
+        content.sound = UNNotificationSound.default
+                        
+        for i in 2...6 {
```

Comment:
I think we could perhaps use constant instead of number? Maybe we can update `dateComponents.hour = 9` as well. What do you think?

Reply:
yeah sure, let me use constants for these values